examples/player/simid_player.html

```diff
@@ -100,10 +121,13 @@
     </table>
     <button onclick="playAd()"> Start ad playback </button>
     <button onclick="initAd()"> Initialize the creative </button>
+    <button onclick="closeAd()"> Close the ad </button> 
```

Comment:
what do you think about only having these show up if an ad is started.  Then could be greyed out or not visible.

Reply:
good idea, fixed this